docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -13,7 +13,7 @@ You must also ensure that your public SSH key is added to the ``authorized_keys`
 
 Continue getting started with Ansible and build an inventory as follows:
 
-#. Create a file named ``inventory.ini`` in the ``ansible_quickstart`` directory.
+#. Create a file named ``inventory.ini`` in the ``ansible_quickstart`` directory that you created in the :ref:`preceding step<get_started_ansible>`.
```

Comment:
```suggestion
#. Create a file named ``inventory.ini`` in the ``ansible_quickstart`` directory that you created in the :ref:`preceeding step<get_started_ansible>`.
```

Reply:
originally i wanted to put it as you're suggesting `preceEding`, but then used Google translate and it showed that the correct word is `preceding`. Are we sure we'll be correct here?

Reply:
@samccann PTAL ^

Reply:
doh! you're right it is precede.. my bad.